src/constants/index.ts

```diff
@@ -221,12 +221,13 @@ export const APP_PATHS = {
   KYBERDAO: '/kyberdao',
   KYBERDAO_STAKE: '/kyberdao/stake-knc',
   KYBERDAO_VOTE: '/kyberdao/vote',
+  KYBERDAO_KNC_UTILITY: '/kyberdao/knc-utility',
   LIMIT: '/limit',
   GRANT_PROGRAMS: '/inter-project-trading-campaigns',
   PROFILE_MANAGE: '/manage',
   ELASTIC_LEGACY: '/elastic-legacy',
   VERIFY_AUTH: '/auth',
-}
+} as const
```

Comment:
why we need it here

Reply:
with:
<img width="1066" alt="image" src="https://github.com/KyberNetwork/kyberswap-interface/assets/23650671/ccbcef36-07a3-4987-ad1b-2c271f6ffbf2">

without:
<img width="1070" alt="image" src="https://github.com/KyberNetwork/kyberswap-interface/assets/23650671/524db2cf-6384-4694-b1d2-ad42b43fe511">